docs/canonicalk8s/metrics/scripts/source_metrics.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+VENV=".sphinx/venv/bin/activate"
+
+files=0
+words=0
+readabilityWords=0
+readabilitySentences=0
+readabilitySyllables=0
+readabilityAverage=0
+readable=true
+
+# measure number of files (.rst and .md), excluding those in .sphinx dir
+files=$(find . -type d -path './.sphinx' -prune -o -type f \( -name '*.md' -o -name '*.rst' \) -print | wc -l)
```

Comment:
same path issue -path `.sphinx`

Reply:
yes, this is  naughty script that should use the values passed to it. will just edit the bash script for now